scripts/integration/axiom/compose.yaml

```diff
@@ -2,12 +2,10 @@ version: '3'
 
 services:
   postgres:
-    image: postgres:${AXIOM_VERSION}
+    image: postgres:${AXIOM_VERSION}-alpine
```

Comment:
Why not keep the `-alpine` as part of the environment matrix? Is it not possible with the existing code, I could envision a need to matrix across different platforms/os/etc of a single version.

Reply:
It's definitely an outlier (for now) and we don't have a good reason (for now...) but if we're always subbing the entire tag, I like that consistency.